climada/hazard/trop_cyclone.py

```diff
@@ -336,46 +336,93 @@ def from_tracks(
 
     def apply_climate_scenario_knu(
         self,
-        ref_year: int = 2050,
-        rcp_scenario: int = 45
+        percentile: str,
+        rcp_scenario: int,
+        target_year: int,
+        baseline: tuple = (1982, 2022)
     ):
         """
-        From current TC hazard instance, return new hazard set with
-        future events for a given RCP scenario and year based on the
-        parametrized values derived from Table 3 in Knutson et al 2015.
-        https://doi.org/10.1175/JCLI-D-15-0129.1 . The scaling for different
-        years and RCP scenarios is obtained by linear interpolation.
-
-        Note: The parametrized values are derived from the overall changes
-        in statistical ensemble of tracks. Hence, this method should only be
-        applied to sufficiently large tropical cyclone event sets that
-        approximate the reference years 1981 - 2008 used in Knutson et. al.
-
-        The frequency and intensity changes are applied independently from
-        one another. The mean intensity factors can thus slightly deviate
-        from the Knutson value (deviation was found to be less than 1%
-        for default IBTrACS event sets 1980-2020 for each basin).
+        From current TC hazard instance, return new hazard set with future events
+        for a given RCP scenario and year based on the parametrized values derived
+        by Jewson 2021 (https://doi.org/10.1175/JAMC-D-21-0102.1) based on those
+        published by Knutson 2020 (https://doi.org/10.1175/BAMS-D-18-0194.1). The
+        scaling for different years and RCP scenarios is obtained by linear
+        interpolation.
+
+        Note: Only frequency changes are applied as suggested by Jewson 2022
+        (https://doi.org/10.1007/s00477-021-02142-6). Applying only frequency anyway
+        changes mean intensities and most importantly avoids possible inconsistencies
+        (including possible counting) that may arise from the application of both
+        frequency and intensity changes, as the relatioship between these two is non
+        trivial to resolve.
 
         Parameters
         ----------
-        ref_year : int
-            year between 2000 ad 2100. Default: 2050
+        percentile: str
+            percentile of Knutson et al. 2020 estimates, representing the model
+            uncertainty in future changes in TC activity. These estimates come from a
+            review of state-of-the-art literature and models. As extreme percentiles,
+            the 5th and 95th are reported for variable cat05 and the 10th and 90th
+            for cat45 and intensity. All three (cat05, cat45, intensity) report the
+            25th, 50th and 75th percentiles. Refer to the mentioned publications above
+            for more details. Possible values are:
+                '5/10', '25', '50', '75', '90/95'
```

Comment:
Can you please add a recommendation for when to use what? This is a rather obscure explanation, and the names `cat05`, `cat45`, `intensity` are unclear here.